src/test/resources/features/RFMStartPage&KBValidations.feature

```diff
@@ -176,9 +176,7 @@ Feature: RFM Start page
     And I click on Continue button
     Then I should see an error message Registration date must be a real date on the RFM Registration Date Page
     When I refresh the page
-    When Registration Day is entered as 10
-    When Registration Month is entered as 10
-    And Registration Year is entered as 2025
+    And I enter future date
     And I click on Continue button
```

Comment:
can we confirm if this is tested in unit/integration testing? If it is, we can probably delete this test

Reply:
I agree with @ridow-hmrc

Reply:
@ridow-hmrc There are no unit test for this error code. So for now I will merge my changes and later when we will have any work around RFM journey we can add it to Unit test and remove it from acceptance tests.